src/components/Undo/Undo.scss

```diff
@@ -0,0 +1,69 @@
+.undo {
+  -moz-box-align: center;
```

Comment:
I wonder if we can switch to the standardized flexbox equivs. for the following three?